### ISSUE TITLE
Only first IPA installed

### ISSUE THREAD

**Issue by Ectalite** [nonymoosee/CydiaImpactorLoader#3]
Hello, 
I use your Loader, I just changed the settings name files, but when i want to install an IPA, the loader just installed the first IPA and not the IPA I chose.
Ectalite

**Comment by nonymoosee** [comment]
You do not need to do anything in settings file.. it should be automatic.. as far as installing the wrong file.. I need more info as far as what is going on.. are they all in the same folder?

**Comment by nonymoosee** [comment]
Are you using the latest version?

**Comment by nonymoosee** [comment]
Try the newest version, selections should be fixed :)

**Comment by Ectalite** [comment]
Thanks a lot, I'm just using 3 different settings files, because I install a lot of IPAs of my iPhone and iPod (and I can just install 3 ipas with 1 apple id).

**Comment by nonymoosee** [comment]
Ah.. ok.. I didn’t know you could only install 3IPAs per account

**Comment by Ectalite** [comment]
What is the differences between the normal and stripped version ?

**Comment by nonymoosee** [comment]
The “stripped” version is a version that is automatically created when using autoit tools if you enable it when you’re compiling.. it goes through your script and “trims the fat” per say.. aka.. a human needs a variable like “$password” when programming to remember that is what you called the variable that you used to save your password. But a computer doesn’t have that limitation.. so it just changed the variables to something short to save on resources.. aka $password gets changed to $p1. Also it goes through and removes any unnecessary libraries that I called but are never used in the program, and also it removes any commentary in the program. If you install the autoit “full install” you can create a “stripped” version for yourself when you compile the script (it’s one of the tabs at the top).

Basically your final exe ends up saving space and using a little less resources.

**Comment by nonymoosee** [comment]
The reason why I uploaded it is so if someone wants to make sure that I didn’t add or manipulate the source code in any way before I compiled it, then they could compile the “stripped” version (x86) and they should in theory get the same checksum, but I haven’t tested this because.. I don’t care since I know it’s clean.
